ci/scripts/unpause-pipeline.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
```

Comment:
When I run `make unpause-pipeline` twice with the same pipeline, the script errors

```bash
❯ make unpause-pipeline
unpaused 'app-autoscaler-release'
error: app-autoscaler-release/no not found on team app-autoscaler

error: app-autoscaler-release/options not found on team app-autoscaler

error: app-autoscaler-release/provided, not found on team app-autoscaler

error: app-autoscaler-release/see not found on team app-autoscaler

error: app-autoscaler-release/`gum not found on team app-autoscaler

error: app-autoscaler-release/choose not found on team app-autoscaler

error: app-autoscaler-release/--help` not found on team app-autoscaler

make: *** [Makefile:7: unpause-pipeline] Error 1

```